crm/main.py

```diff
@@ -520,9 +553,19 @@ def nexthop(ctx):
     elif ctx.obj["crm"].cli_mode == 'resources':
         ctx.obj["crm"].show_resources('{0}_nexthop'.format(ctx.obj["crm"].addr_family))
 
+@mpls.command()
+@click.pass_context
+def inseg(ctx):
+    """Show CRM information for in-segment resource"""
+    if ctx.obj["crm"].cli_mode == 'thresholds':
+        ctx.obj["crm"].show_thresholds('{0}_inseg'.format(ctx.obj["crm"].addr_family))
+    elif ctx.obj["crm"].cli_mode == 'resources':
+        ctx.obj["crm"].show_resources('{0}_inseg'.format(ctx.obj["crm"].addr_family))
+
```

Comment:
I didn't understand why this is a repeat of line 520-528

Reply:
@smaheshm This cut-n-paste remnant has been removed.